pyrolite/geochem/alteration.py

```diff
@@ -127,3 +127,26 @@ def WIP(df: pd.DataFrame):
 
     """
     return 2 * df.Na2O / 0.35 + df.MgO / 0.9 + 2 * df.K2O / 0.25 + df.CaO / 0.7
+
+def AIOCG_xy(df: pd.DataFrame):
```

Comment:
I think this function should probably return a dataframe with appropriately labeled columns (`["(2Ca+5Fe+2Mn)+(2Ca+5Fe+2Mn+Mg+Si)", "K/(K+Na+0.5Ca)"])` so it's really clear what's being returned (perhaps this explicit naming should be the same for the other alteration functions..!). 

I've recently updated `develop` with the other alteration functions so this will likely conflict unless you want to pull recent changes from there; we'll need to resolve them before merging this in. The whitespace at the start of the docstring will also need to be removed here, and brief sections for `Parameters` and `Returns` can be added.

I think it might also be good to separate the '400 x 600` dimensions of the diagram from the molar proportions unless there's some reason to include them? We can set the aspect of an axis separately from the data coordinates - which I think vary from 0-1, going off the original citation?